src/engine/behavior_script.c

```diff
@@ -928,11 +927,13 @@ void cur_obj_update(void) {
     // Handle visibility of object
     if (o->oRoom != -1) {
         // If the object is in a room, only show it when Mario is in the room.
-        if (
-            (objFlags & OBJ_FLAG_ACTIVE_FROM_AFAR)
-            || distanceFromMario < o->oDrawingDistance
-        ) {
-            cur_obj_enable_disable_room_rendering(inRoom);
+        if ((objFlags & OBJ_FLAG_ACTIVE_FROM_AFAR)
+         || distanceFromMario < o->oDrawingDistance) {
```

Comment:
try not to do these kinds of formatting changes until we have some kind of style guide, personally i dont think this looks good at all, whereas the previous formatting seems more legible to me. either way its something we should plan for later.

Reply:
I've been using the first style more often recently for multi-line if statements because it's a lot nore clear what goes on inside of the block (and this comes from somebody that hates allman style formatting)

Reply:
Reverted